model/AI/Properties/typeOfModel.md

```diff
@@ -0,0 +1,17 @@
+SPDX-License-Identifier: Community-Spec-1.0
+
+# typeOfModel
+
+## Summary
+
+Field records the type of the model used in the AI software.  
+
+## Description
+
+This is a text field that records the type of the AI model(s) used in the software. For instance if its supervised model, unsupervised model or if its a reinforcement learning model or a combination. Preferably expressed using AND conjugation if there are multiple models being used as a part of the AI software. 
+
+## Metadata
+
+- name: typeOfModel
+- Nature: DataProperty
+- Range: xsd:string
```

Comment:
Another possible future enumeration

Reply:
Discussed with Karen:  Yes, in future, but for now, we need to collect the data.